### DISCUSSION TITLE
is possible to runOnInit with gstream?

### DISCUSSION THREAD

**Discussion by SlowMonk** [bluenviron/mediamtx#2047] [Questions]
` #runOnInit: ffmpeg -f v4l2 -i /dev/video0 -pix_fmt yuv420p -preset ultrafast -b:v 600k -f rtsp rtsp://10.10.10.68:$RTSP_PORT/test
`  
with ffmpeg it work fine 
  
  runOnInit: gst-launch-1.0 -v v4l2src device=/dev/video0 ! videoconvert ! video/x-raw,format=I420 ! x264enc tune=zerolatency bitrate=1024 speed-preset=ultrafast ! h264parse ! rtph264pay config-interval=1 pt=96 ! rtspclientsink location=rtsp://10.10.10.68:8554/test

try to change it with gst. 
but it gives following errors.

```
(gst-launch-1.0:2888527): GStreamer-WARNING **: 13:40:16.121: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstrtspclientsink.so': /usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstrtspclientsink.so: undefined symbol: gst_rtsp_stream_transport_set_list_callbacks
WARNING: erroneous pipeline: no element "rtspclientsink"
2023/07/11 13:40:16 INF [path cam] runOnInit command exited: command returned code 1
```

**Comment by aler9** [comment]
duplicate of #2046
